doc/api.rst

```diff
@@ -254,48 +277,74 @@ Functions for in the :mod:`plts` module, for plotting time series and analysis o
 Time Series
 ~~~~~~~~~~~
 
-.. currentmodule:: neurodsp.plts.time_series
+.. currentmodule:: neurodsp.plts
 
 .. autosummary::
-  :toctree: generated/
+    :toctree: generated/
 
-  plot_time_series
-  plot_instantaneous_measure
-  plot_bursts
+    plot_time_series
+    plot_instantaneous_measure
+    plot_bursts
 
 Spectral
 ~~~~~~~~
 
-.. currentmodule:: neurodsp.plts.spectral
+.. currentmodule:: neurodsp.plts
 
 .. autosummary::
-  :toctree: generated/
+    :toctree: generated/
 
-  plot_power_spectra
-  plot_scv
-  plot_scv_rs_lines
-  plot_scv_rs_matrix
-  plot_spectral_hist
+    plot_power_spectra
+    plot_scv
+    plot_scv_rs_lines
+    plot_scv_rs_matrix
+    plot_spectral_hist
 
 Filter
 ~~~~~~
 
-.. currentmodule:: neurodsp.plts.filt
+.. currentmodule:: neurodsp.plts
 
 .. autosummary::
-  :toctree: generated/
+    :toctree: generated/
 
-  plot_filter_properties
-  plot_frequency_response
-  plot_impulse_response
+    plot_filter_properties
+    plot_frequency_response
+    plot_impulse_response
 
 Rhythm
 ~~~~~~
 
-.. currentmodule:: neurodsp.plts.rhythm
+.. currentmodule:: neurodsp.plts
 
 .. autosummary::
-  :toctree: generated/
+    :toctree: generated/
+
+    plot_swm_pattern
+    plot_lagged_coherence
+
+
+Fetch/Load Data
```

Comment:
These are only really for test data files used in the tutorials. I don't think they really need to be listed in the API, as it sortar suggests they might be more broadly relevant and useful than they are.

Reply:
I only added them in so they would link in the tutorials. I'll go ahead a drop them.